internal/utils/docker.go

```diff
@@ -184,7 +184,7 @@ func GetRegistryAuth() string {
 }
 
 // Defaults to Supabase public ECR for faster image pull
-const defaultRegistry = "public.ecr.aws"
+const defaultRegistry = "docker.io"
```

Comment:
To be changed. Currently this version of logflare isn't available in ecr

Reply:
This needs to remain the previous value, we need to publish a mirror image of logflare to ecr. For testing you can use this environment variable at runtime to change the registry `SUPABASE_INTERNAL_IMAGE_REGISTRY=docker.io`.

example:
```sh 
SUPABASE_INTERNAL_IMAGE_REGISTRY=docker.io supabase start
```

Reply:
Ah nice one! Will change to use the env var for now until we push the image 🙏